Header header
uint64 start_time
Object[] objects
